projects/desktop/src/constants/validators.ts

```diff
@@ -0,0 +1,14 @@
+export const required = (value: string): string | undefined  => {
```

Comment:
`export const required = (value: string): string | undefined  => 
    value ? undefined : 'fieldError.required'
`

Check if it works like this